src/main/java/com/pkb/unit/Unit.java

```diff
@@ -109,7 +109,8 @@ public Unit(String id, Bus bus, long retryPeriod, TimeUnit retryTimeUnit) {
 
 
     /**
-     * @return true if this set did not already contain the specified element
+     * Adds a dependency to this unit.
+     * This unit can only start if all it's dependencies are started.
```

Comment:
Sorry Martin for the very detailed remark but isn't it "... if all its dependencies are started"?
https://dictionary.cambridge.org/grammar/british-grammar/pronouns/pronouns-possessive-my-mine-your-yours-etc

Reply:
hot damn you're right

Reply:
I'll fix in the web editor!

Reply:
Cool :)